src/zoid/buttons/util.js

```diff
@@ -222,3 +223,58 @@ export function applePaySession() : ?ApplePaySessionConfigRequest {
         return undefined;
     }
 }
+
+export function getButtonSize(props : ButtonProps, container : string | HTMLElement | void) : string | void {
+    if (!container) {
+        return;
+    }
+
+    let containerWidth = 0;
+
+    if (typeof container !== 'string') {
+        containerWidth = getElement(container)?.offsetWidth;
+    }
+
+    if (typeof container === 'string') {
+        const containerElement = document.querySelector(container);
+        containerWidth = containerElement.offsetWidth || 0;
```

Comment:
I'm thinking we need to ensure `containerElement` is truthy before getting the `offsetWidth` prop.
```suggestion
        containerWidth = containerElement?.offsetWidth || 0;
```